### PR TITLE
Automate video dashboard screen

### PR DESCRIPTION
**Pull request by bilalawan321**

### CODE REVIEW

**Comment by bilalawan321** [comment]
@jawad-khan can you please review this PR.?

File: tests/android/tests/test_android_videos_dashboard.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        android_my_courses_list_page = AndroidMyCoursesList(set_capabilities, setup_logging)
+        android_main_dashboard_page = AndroidMainDashboard(set_capabilities, setup_logging)
+
+        assert android_main_dashboard_page.load_courses_tab()
+        if android_my_courses_list_page.get_my_courses_list_row():
+            android_my_courses_list_page.get_first_course().click()
+        else:
+            setup_logging.info('No course enrolled by this user.')
+
+    def test_load_contents_smoke(self, set_capabilities, setup_logging):
```
Comment (jawad-khan) [inline tests/android/tests/test_android_videos_dashboard.py:44]: This function is too long, can we break this into smaller functions?

**Review by jawad-khan — APPROVED** [review]

File: tests/android/tests/test_android_videos_dashboard.py (hunk 1 above)
Comment (bilalawan321) [reply tests/android/tests/test_android_videos_dashboard.py:44]: this method name is the same in all files, 
if we change in this file, then we have to change in every test file.

Comment (bilalawan321) [reply tests/android/tests/test_android_videos_dashboard.py:44]: I have split functions data into two functions.

**Pull request merged**